src/components/StatusBar/StatusBarToggleButton.tsx

```diff
@@ -0,0 +1,60 @@
+import {
```

Comment:
for the use of `React` below:
```suggestion
import React from "react";

import {
```

(TypeScript will implicitly import `React` even if we don't specify such, but it's always better to be explicit

Reply:
Fixed!